src/tests/ftest/util/pool_security_test_base.py

```diff
@@ -95,42 +73,64 @@ def get_pool_acl_list(self, uuid):
                     pool_permission_list.append(line)
         return pool_permission_list
 
-    def update_pool_acl_entry(self, uuid, action, entry):
+    def update_pool_acl_entry(self, action, entry):
         """Update daos pool acl list by dmg tool.
 
         Args:
-            uuid (str): pool uuid.
             action (str): update-acl or delete-acl.
             entry (str): pool acl entry or principal to be updated.
-
-        Return:
-            none.
-
         """
         if action == "delete":
-            result = self.dmg.pool_delete_acl(uuid, entry)
+            result = self.pool.delete_acl(entry)
         elif action == "update":
-            result = self.dmg.pool_update_acl(uuid, None, entry)
+            result = self.pool.update_acl(use_acl=False, entry=entry)
```

Comment:
(pylint-assignment-from-no-return) Assigning result of a function call, where the function has no return